packages/chain-adapters/src/utxo/bitcoin/BitcoinChainAdapter.test.ts

```diff
@@ -547,4 +547,56 @@ describe('BitcoinChainAdapter', () => {
       expect(res).toMatchObject(expectedReturnValue)
     })
   })
+
+  describe('getBIP44Params', () => {
+    const adapter = new bitcoin.ChainAdapter(args)
+    it('should throw for undefined accountType', async () => {
+      expect(() => {
+        adapter.getBIP44Params({ accountNumber: 0, accountType: undefined })
+      }).toThrow('not a supported accountType undefined')
+    })
+    it('should always be coinType 0', async () => {
+      for (const acctType of adapter.getSupportedAccountTypes()) {
```

Comment:
`F2` and rename symbol `acctType` -> `accountType` then we can use object shorthand?